_components/sidenav.md

```diff
@@ -2,13 +2,13 @@
 layout: styleguide
 type: component
 title: Side navigation
-lead: "Hierarchical, vertical navigation to place at the side of a page. Note: we are currently developing horizontal navigation and headers for the top of a page"
+lead: Hierarchical, vertical navigation to place at the side of a page. Note: We're currently developing horizontal navigation and headers for the top of a page.
```

Comment:
@awfrancisco we need quotes here, bc jekyll requires this in frontmatter when there's a `:`. cc @carodew